org-configurations/default-configuration.yml

```diff
@@ -217,15 +217,15 @@ client:
             icon: "twitter.svg"
             url: "#"
       input_fields:
-        username:
-          type: "text"
-          pattern: '[a-zA-Z@.+\-_\d]{1,150}'
+        email:
+          type: "email"
+          pattern: null
```

Comment:
I think there should be a pattern check for emails as well.
@nikitaermishin any specific reason why we are not doing pattern check like `<something>@<somthing>.<something>` ?

Reply:
@nikitaermishin Ah, I didn't factor compatibility, please proceed whatever is best for the compatilibity! :smile:

Reply:
@atb00ker So, the best for compatibility will be to use both of them (if `pattern` is not supported for some very old browsers (like `Chrome 5`) it will ignore `pattern` and only use `type="email"`). It will provide good compatibility and also good validation, so I think it is what we're looking for :smile:

#### Compatibility tables:
 - [pattern](https://caniuse.com/#feat=input-pattern)
 - [type="email"](https://caniuse.com/#feat=input-email-tel-url)